source/more-info/unhealthy/oserror_bad_message.markdown

```diff
@@ -0,0 +1,17 @@
+---
+title: "Operating System error: Bad message"
+description: "More information on what to do when OS errors mark the installation as unhealthy."
+---
+
+## The issue
+
+The Supervisor received Operating System errors during file operation (Python OSError exceptions). The error number reported was 74, "Bad message", which is typically raised on file system corruption.
+
+## The solution
+
+File system corruption can happen due to power outage or when the system otherwise did not get shutdown properly. Typically Home Assistant recovers from file system corruption, so if you see this error the first time consider just ignoring it. Just in case the problem gets worse, make sure to create a full backup (and download it or store it on a different system).
```

Comment:
```suggestion
Typically, Home Assistant recovers from file system corruption. If you see this error for the first time, ignore it. In case the problem gets worse, make sure to create a full backup, download it, or store it on a different system.
```